packages/landing-gatsby/src/containers/App/ProcessList/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import 'rc-tabs/assets/index.css';
+import Container from 'common/src/components/UI/Container';
+import PropTypes from 'prop-types';
+import { useStaticQuery, graphql } from 'gatsby';
+import Text from 'common/src/components/Text';
+import Heading from 'common/src/components/Heading';
+import Box from 'common/src/components/Box';
+import ListItemWrapper, {
+  SectionWrapper,
+  CardBg,
+  Icon,
+  IconWrapper,
+  ContentWrapper,
+} from './processList.style';
+
+const ProcessList = ({
+  title,
+  description,
+  icon1,
+  icon2,
+  ListElementWrapper,
+  arrow,
+}) => {
+  const Data = useStaticQuery(graphql`
+    query {
+      allProcessesJson(limit: 10) {
+        edges {
+          node {
+            id
+            summary {
+              abstract
+            }
+            icons {
+              endIconUrl
+              startIconUrl
+            }
+            title
+          }
+        }
+      }
+    }
+  `);
+  return (
+    <SectionWrapper>
+      <Container>
+        <CardBg>
+          <Box>
+            {Data.allProcessesJson.edges.map((process, index) => (
+              <ListItemWrapper {...ListElementWrapper} key={index}>
+                <IconWrapper className="icon__wrapper">
+                  <Icon
+                    src={process.node.icons.startIconUrl}
+                    alt={process.node.title}
+                    {...icon1}
+                  />
+                  <Text content=">>>" {...arrow} />
+                  <Icon
+                    src={process.node.icons.endIconUrl}
+                    alt={process.node.title}
```

Comment:
this could be read from `process.node.instructions[process.node.instructions.length-1].tool.name`

Reply:
thanks